src/main/java/com/streisky/virtualshowcasebackend/config/security/AuthenticationService.java

```diff
@@ -0,0 +1,20 @@
+package com.streisky.virtualshowcasebackend.config.security;
+
+import com.streisky.virtualshowcasebackend.repository.account.AccountRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.stereotype.Service;
+
+@Service
+public class AuthenticationService implements UserDetailsService {
```

Comment:
I will keep this name because this service is used for authentication only.

If a service is needed for acount in the future, it will have its own service and will not mix business rules with authentication or confuse with the same names.